corehq/form_processor/serializers.py

```diff
@@ -178,7 +178,7 @@ class CommCareCaseSQLSerializer(DeletableModelSerializer):
 
     class Meta(object):
         model = CommCareCaseSQL
-        exclude = ('id', 'case_json',)
+        exclude = ('id',)
```

Comment:
@snopoke @czue might be helpful to just get your confirmation on my commit comment for 963bcc7, copied here for easier back and forth:

> It looks like this exclude was being ignored in djangorestframework 3.5
> but that some later versions check to make sure it's either in exclude
> or set as a property.
> 
> Digging back to how we got here, it looks like it was two PRs @czue and
> @snopoke worked on separately but at the same time and in communication
> with each other: @czue's https://github.com/dimagi/commcare-hq/pull/10560/files#diff-386b068280d3d01753890cf76a65da50R46
> and @snopoke's https://github.com/dimagi/commcare-hq/pull/10581/files#diff-386b068280d3d01753890cf76a65da50R50.
> It looks like @czue's change to add the case_json property with an
> explicit type won out over @snopoke's to exclude that property,
> but both ended up in the code. I'm guessing no one noticed because even
> though both ended up in the code, it happened to have the intended
> result anyway.

Does that sound about right?

Reply:
Sorry, I have literally no recollection of any of this but your story sounds/looks right to me.

Just confirming my understanding: the result in DRF 3.5 of the two changes was that `case_json` is included, but DRF 3.11 validates the situation and declares it illegal? So you're just removing the line that was previously being ignored anyways?

Reply:
That's my understanding too. In addition to travis tests passing, I checked that a sample regular form and case API each look the same before and after the changes in this PR, so that matches up with that mental model

Reply:
looks right to me